### DISCUSSION TITLE
"Unknown repo" for existing repo

### DISCUSSION THREAD

**Discussion by karlhorky** [community/community#13888] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi there, thanks for working on improving the GitHub search experience, really cool!

One thing that I noticed is that it's not possible to search a user's repos, eg:

```
repo:Alexou-riri/final-project-bootcamp @media
```

I get the error that this is not a known repo:

```
Unknown repo: "Alexou-riri/final-project-bootcamp"
```

But it is (at least, at the time of writing): https://github.com/Alexou-riri/final-project-bootcamp

**Comment by colinwm** [comment, marked as answer]
Hey @karlhorky, not all repositories are indexed in the new cs.github.com search. 

We've indexed many popular open source repositories, plus all of the repositories owned by people who have signed up for the technology preview. Seems like `Alexou-riri` probably hasn't signed up yet, so their repos aren't indexed!

**Reply by karlhorky** [reply]
Ah ok, missed this detail, sorry! Thanks for the quick answer!

**Reply by gnu4cn** [reply]
Please give me the old search tool back, I even can not search my own repo.

**Comment by eherrerosj** [comment]
@colinwm, is there a plan in place to index all code repositories present in GitHub?

**Reply by colinwm** [reply]
Yes, we intend to index everything soon (hopefully order of a few months from now).

But in the mean time, if you search inside a specific repo on github.com (e.g. `repo:my/repo xyz`) which is not yet indexed, we will immediately queue that repo to be indexed. BTW, this feature only works if you have the new code search in github.com beta enabled.

**Reply by gnu4cn** [reply]
I would honestly say, you microsoft guys made the github messedup more and more.

**Comment by JinchaoLove** [comment]
@colinwm What if we want to search in a new repository? How to disable the new code search in github.com beta? I just need the basic search abilities.

**Reply by look** [reply]
We're working on improving the new repository indexing. For now, please do a search like `repo:your/repo` and it will make the repository searchable within about 5 minutes.

**Reply by JinchaoLove** [reply]
Dear @look, I tried searching and waiting, but after several hours later, it still not works.

<img width="896" alt="image" src="https://github.com/community/community/assets/34153355/ebc4868d-204c-47f3-8233-231f41ca6c62">

**Reply by look** [reply]
Thanks @JinchaoLove. I'm looking into this.

**Reply by look** [reply]
Your repository was not indexed due to a bug with handling repositories that were deleted and then recreated. We are rolling out a fix for that, sorry for the delay.

**Reply by HenryPook** [reply]
Hi @look,

I'm also experiencing this issue with https://github.com/pooky-lighting/pooky-shopify-theme. I've tried to search but days later it's still not indexed. 

Could you take a look at this for us please?

Thanks!

**Reply by look** [reply]
@HenryPook I've indexed your `pooky-lighting` repositories. I'm not sure why indexing-on-search didn't work, did you happen to record the message you saw?

**Reply by HenryPook** [reply]
Hi @look,

Thanks very much, we use the feature a lot so I appreciate you indexing them. 

It was the same error that other users have noticed here - 'Unknown repo: "repo name". I tried searching a few hours and then a few days later after initial searches but it kept coming up with the same error.

We recently added a number of new repos, one of which was searched, they were existing repos that were ported to our account as a new remote, so I wonder if that had something to do with it?

**Reply by roke-julian-lockhart** [reply]
@look, ~~this persists at [`https://github.com/search?q=repo%3Aarancaytar%2Faran-web+path%3A%2F%5Esource%5C%2Fconverters%5C%2Ftime%5C%2F%2F+b36&type=code#:~:text=Save-,Unknown%20repo,-:%20%22arancaytar/aran%2Dweb%22`](https://github.com/search?q=repo%3Aarancaytar%2Faran-web+path%3A%2F%5Esource%5C%2Fconverters%5C%2Ftime%5C%2F%2F+b36&type=code#:~:text=Save-,Unknown%20repo,-:%20%22arancaytar/aran%2Dweb%22).~~ [^1] The error message is somewhat nonsensical:

<p></p>

<img width="565" height="84" alt="image" src="https://github.com/user-attachments/assets/e990aa3e-d15a-4a0f-95de-3ae23a2707e8" /> [^3]

<p></p>

#### As Of `T18:47:39+00:00`

It's been remediated:

<p></p>

<img width="1276" height="1076" alt="@{&#39;Date&#39;=&#39;20260125T184819+0000&#39;; &#39;Type&#39;=&#39;Photo&#39;; &#39;Origin&#39;=&#39;Code search results — Mozilla Firefox&#39;}#" src="https://github.com/user-attachments/assets/7bb436bb-b2b4-4f9f-a110-fd2434787a8e" /> [^3]

<p></p>

…although the error message should be improved regardless.

[^1]: [`stackoverflow.com/questions/73941304/how-to-search-github-repository-at-a-specific-commit#comment131148130_73967172`](https://stackoverflow.com/questions/73941304/how-to-search-github-repository-at-a-specific-commit#comment131148130_73967172) [^2]

[^2]: [`stackoverflow.com/questions/20739562/converting-large-numbers-into-letters-and-back-again#comment140957090_20750581`](https://stackoverflow.com/questions/20739562/converting-large-numbers-into-letters-and-back-again#comment140957090_20750581)

[^3]: [`github/markup/issues/2005`](https://github.com/github/markup/issues/2005#issue-3553990404)

**Comment by magudeshhmw** [comment]
✅ What the problem means

When you searched like this:

repo:Alexou-riri/final-project-bootcamp @media


GitHub Code Search returned:

Unknown repo: "Alexou-riri/final-project-bootcamp"

Even though the repository actually exists.

✅ Why this happens

At the time of that discussion (2022), GitHub’s new Code Search (cs.github.com) did NOT index all repositories.

GitHub indexed only:

Popular open-source repositories

Repositories owned by users who signed up for the Code Search technology preview

So if a user (like Alexou-riri) hadn’t joined the preview, their repos were not indexed, meaning:

The repo exists ✅

But Code Search can’t search it ❌

That’s why it showed “Unknown repo”.

✅ Important clarification

This issue was specific to the new GitHub Code Search, not normal GitHub search.

Normal GitHub search → can find any public repo

Code Search (cs.github.com) → only indexed repos

✅ What about now?

Today, GitHub Code Search has mostly rolled out globally, so:

Most public repositories are indexed

But some private or restricted repos may still not be searchable

✅ Simple analogy

Think of Code Search like Google Search:

Website exists → doesn’t mean Google has indexed it yet.

**Reply by roke-julian-lockhart** [reply]
@Magudeshhmw, don't respond with regurgitated LLM output:

> Your report has been submitted for review.

**Reply by mecodeatlas** [reply]
Hi @Magudeshhmw, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. 

Thank you for helping us maintain an authentic and beneficial space for everyone.
